tests/js/jest.config.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
 			'@wordpress/jest-preset-default/scripts/setup-globals.js'
 		),
 	],
+	globals: {
+		__PAYMENT_METHOD_FEES_ENABLED: false,
```

Comment:
The flag needs to be configured as a global variable for Jest to recognize - setting the default value here